### PR TITLE
Added: Frame variant of cover in resume component

### PR DESCRIPTION
**Pull request by rahulgoyal21**
Frame variant of cover in resume component has been added.
Screenshot from 2020-08-02 13-38-46
Screenshot from 2020-08-02 13-39-42

### CODE REVIEW

File: src/modules/resume/cover/variants/frame/render.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  imageUrl,
+  githubUrl,
+  linkedinUrl
+}) => {
+  const styles = useStyles(theme);
+
+  return (
+    <div
+      style={{
+        ...styles.root
```
Comment (AKAspanion) [inline src/modules/resume/cover/variants/frame/render.js:74]: unnecessary spread.

File: src/modules/resume/cover/variants/frame/render.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+      }}
+    >
+      <div style={styles.outerBoxStyle}>
+        <div style={styles.design}>
+          <img src={imageUrl.value} alt="Rahul Goyal" style={styles.imgStyle} />
```
Comment (AKAspanion) [inline src/modules/resume/cover/variants/frame/render.js:79]: Use a dynamic value for alt.

File: src/modules/resume/cover/variants/frame/render.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+            <div style={styles.dataWhite}></div>
+            <div></div>
+            <div></div>
+            <div style={styles.dataWhite}></div>
+          </div>
+
+          <div style={styles.blend}>
+            <img
+              src={imageUrl.value}
+              alt="Rahul Goyal"
```
Comment (AKAspanion) [inline src/modules/resume/cover/variants/frame/render.js:98]: Don't use your name for alt.

File: src/modules/resume/cover/variants/frame/render.js [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+              justifyContent: 'center',
+              alignItems: 'center'
+            }}
+            href={linkedinUrl.value}
+            target="_blank"
+            referrer="noreferrer"
+          >
+            <img
+              src={linkedinIcon}
+              style={{ borderRadius: '50%', width: '48px', height: '48px' }}
```
Comment (AKAspanion) [inline src/modules/resume/cover/variants/frame/render.js:148]: Move these styles to styles object.

Comment (rahulgoyal21) [reply src/modules/resume/cover/variants/frame/render.js:148]: Done

File: src/modules/resume/cover/variants/frame/render.js (hunk 1 above)
Comment (rahulgoyal21) [reply src/modules/resume/cover/variants/frame/render.js:74]: Done

File: src/modules/resume/cover/variants/frame/render.js (hunk 3 above)
Comment (rahulgoyal21) [reply src/modules/resume/cover/variants/frame/render.js:98]: Done

File: src/modules/resume/cover/variants/frame/render.js (hunk 2 above)
Comment (rahulgoyal21) [reply src/modules/resume/cover/variants/frame/render.js:79]: Done

**Pull request merged**